### ISSUE TITLE
[a11y] Insufficient color contrast issue on /firefox/all/ page

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15060]
## Description

The grayed out steps on the /firefox/all/ page lack sufficient color contrast:

![image](https://github.com/user-attachments/assets/fa4bd93b-731c-42ef-8f0b-e2ee23bb660c)

### Steps to reproduce

https://www-dev.allizom.org/en-US/firefox/all/

### Expected result

No color contrast issues

### Actual result

![image](https://github.com/user-attachments/assets/bb3da986-7422-4df6-abfc-9c79350d6a7c)

### Environment

N/A

**Comment by reemhamz** [comment]
From WCAG:

> Incidental: Text or images of text that are part of an inactive [user interface component](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#user-interface-componentdef), that are [pure decoration](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#puredecdef), that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

Since the greyed out text is technically an inactive component until the user finished the previous selections, we can probably put aside the color-contrast issue here. However, that doesn't mean we should ignore this entirely. I noticed that on the DOM, the greyed out section doesn't have the appropriate `aria-disabled` attributes that could help screenreader users. I created an issue for that here https://github.com/mozilla/bedrock/issues/15065.

We could also probably add a `cursor: not-allowed` declaration to help with visual indication.

I'd be happy to tackle both of these issues in one PR. Thoughts on the suggestions above?

**Comment by alexgibson** [comment]
Sounds good to me Reem - the tests may be happy if it understands the elements are supposed to be in a disabled state 👍

**Comment by alexgibson** [comment]
From a user perspective though, it does seem like this text should still have some level of sufficient contrast, so people can at least understand that there are additional steps to come?

**Comment by reemhamz** [comment]
Agreed, I was playing around with the opacity (since the text is colored via an opacity declaration rather than a hex) and I think it'd be a good idea to also choose a slightly darker color. I'll add that to the tasks! 👍🏼
